tmt/steps/report/reportportal.py

```diff
@@ -142,6 +166,17 @@ class ReportReportPortalData(tmt.steps.report.ReportStepData):
              (e.g. 'Idle'). 'To Investigate' is used by default.
              """)
 
+    log_size_limit: int = field(
+        option="--log-size-limit",
+        metavar="LOG_SIZE_LIMIT",
```

Comment:
Shouldn't this be just `SIZE` or `LIMIT` or `BYTES` or something similar?

Reply:
I didn't comment on this because `reportportal` docstring & options help texts are in a queue for complete review, including metavars. So, getting one right didn't look important to me when the rest needed to be polished too.

Reply:
In fact, I am considering introducing one more (smaller) limit specifically for traceback file.

Reply:
@kkaarreell, so do you still plan some changes here? Will you add the release note / schema adjustment? Or shall I?

Reply:
Hi @psss , I have just pushed one more commit as I have decided to separate ordinary test logs and error/traceback logs that are being used for auto-analysis. I believe such a log should be even smaller.

Reply:
I belive that would be `tmt/schemas/report/reportportal.yaml` and `docs/releases.rst`